scripts/processTranslations.js

```diff
@@ -0,0 +1,56 @@
+const fs = require('fs')
```

Comment:
What's the reason this js (instead of ts)?

Reply:
I always have configuration issues trying to run typescript in standalone files like this, I'm trying now and running into errors, looking into how to fix them

Reply:
Done! :)